dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -882,6 +902,266 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
 
     }
 
+    private void fillInMetadata(DepositMetadata depositMetadata, Workflow workflow, WorkflowVersion workflowVersion) {
```

Comment:
These methods are a bit big, please move them out to a utility class akin to GoogleHelper or EntryLabelHelper

Reply:
Done